docs/REST API/rest-api.md

```diff
@@ -980,7 +980,14 @@ None.
     }
   },
   "integrations": {
-    "jira": null
+    "jira": null,
+    "google_calendar": [
```

Comment:
is there a reason this is an array and not an object? @rachaelshaw @getvictor

Reply:
We will support multiple configs in the future. For example, one company acquires another, so they are on different Google Workspace accounts and with different domains.

Reply:
What will "not configured" look like? `[]`? `null`?

Reply:
For global configs it will be `[]` or `null` or `[null]`, for team config it will be `null`

Reply:
thank you!